packages/backend/src/utils/parsers/quadlet-dryrun-parser.spec.ts

```diff
@@ -72,6 +72,97 @@ quadlet-generator[13914]: converting "nginx.image": unsupported key 'Annotation'
 converting "nginx2.image": unsupported key 'jhfhfhf' in group 'Image' in /home/user/.config/containers/systemd/nginx2.image
 `;
 
+const TEMPLATE_QUADLET: string = `
```

Comment:
Suggestion: move these objects in another file because the tests only start after 150 lines

Reply:
https://github.com/podman-desktop/extension-podman-quadlet/pull/752/commits/f474e850c1c7f5757d62d1429e47f47fe45b2e3e